packages/salesforcedx-vscode-soql/src/editor/soqlEditorInstance.ts

```diff
@@ -223,6 +247,10 @@ export class SOQLEditorInstance {
   protected async retrieveSObjects(): Promise<void> {
     return withSFConnection(async conn => {
       conn.describeGlobal$((err, describeGlobalResult) => {
+        if (err) {
+          const message = nls.localize('error_sobjects_request');
+          channelService.appendLine(message);
+        }
```

Comment:
There will be conflicts with @dobladez 's [PR ](https://github.com/forcedotcom/salesforcedx-vscode/pull/2792) as these methods have been refactored.  But the message will likely remain the same and simply appear in the channel.